### PR TITLE
missing function 'indexOf', fixed solvePath, added test

### PR DESCRIPTION
**Pull request by jondot**

### CODE REVIEW

**Comment by Yomguithereal** [comment]
Thanks for the fix @jondot!

File: test/suites/helpers.js [hunk 1]
```diff
@@ -51,6 +51,24 @@ describe('Helpers', function() {
     });
   });
 
+  describe('Solve path', function() {
+
+    it('should be able to solve a complex path', function() {
+      o = {
```
Comment (Yomguithereal) [inline test/suites/helpers.js]: Is this variable supposed to be global?

Comment (jondot) [inline test/suites/helpers.js]: no, sorry.

**Comment by Yomguithereal** [comment]
Thanks, merging right now!

**Pull request merged**